src/core/lib/layer.js

```diff
@@ -136,13 +136,26 @@ export default class Layer {
     if (changeFlags.dataChanged) {
       this.invalidateAttribute('all');
     }
+    if (changeFlags.propsChanged) {
```

Comment:
I think this can be handled more efficiently, we could just extract it from the props object that is already supplied to AttributeManager when we trigger an accessor.

As stated above, my current thinking is that `props.transition` should be ignore and never trigger a prop update (as otherwise it will always trigger an update or require an expensive deep equal or apps to declare `attribute` as global constants.)

Given that we have an open PR on not triggering a change on inline functions, we may want a table that describes how different props are diffed.